api/src/main/resources/notebooks.yaml

```diff
@@ -1442,10 +1439,7 @@ definitions:
         default: false
       customClusterEnvironmentVariables:
         type: object
-        description: A collection of key/value pairs of environment variable names and their desired value to be set in the cluster.
-        additionalProperties:
-          type: string
-
+        description: Optional environment variables to be set on the cluster.
```

Comment:
This seems less descriptive than previously, although does align with Leo's. Ours appears to have always been different than Leo's in this case (looking at history).

I'd probably prefer to keep our more descriptive description and PR Leo with the updated one? Was there a reason to update this, or was it just copying their most recent swagger?

Reply:
This was strictly a copy.  We can go the other way.

Reply:
Okay, I'd leave our old description around then on this one, and PR leo with that. (Mainly because it seems mostly unrelated to the change here.)